nemo/collections/nlp/modules/common/megatron/megatron_init.py

```diff
@@ -149,12 +155,14 @@ def initialize_model_parallel_for_nemo(
     set_expert_model_parallel_world_size(app_state.expert_model_parallel_size)
     set_expert_model_parallel_rank(app_state.expert_model_parallel_rank)
 
+    set_pipeline_model_parallel_world_size(
+        app_state.pipeline_model_parallel_size + app_state.encoder_pipeline_model_parallel_size
+    )
```

Comment:
Okay so `pipeline_model_parallel_size` is really the decoder pipeline size now?

Reply:
yep